tests/contenttypes_tests/test_operations.py

```diff
@@ -74,9 +74,36 @@ def test_missing_content_type_rename_ignore(self):
     def test_content_type_rename_conflict(self):
         ContentType.objects.create(app_label='contenttypes_tests', model='foo')
         ContentType.objects.create(app_label='contenttypes_tests', model='renamedfoo')
-        call_command('migrate', 'contenttypes_tests', database='default', interactive=False, verbosity=0)
+        call_command('migrate', 'contenttypes_tests', '0002', database='default', interactive=False, verbosity=0)
         self.assertTrue(ContentType.objects.filter(app_label='contenttypes_tests', model='foo').exists())
         self.assertTrue(ContentType.objects.filter(app_label='contenttypes_tests', model='renamedfoo').exists())
         call_command('migrate', 'contenttypes_tests', 'zero', database='default', interactive=False, verbosity=0)
         self.assertTrue(ContentType.objects.filter(app_label='contenttypes_tests', model='foo').exists())
         self.assertTrue(ContentType.objects.filter(app_label='contenttypes_tests', model='renamedfoo').exists())
+
+
+@override_settings(
+    MIGRATION_MODULES=dict(
+        settings.MIGRATION_MODULES,
+        contenttypes_tests='contenttypes_tests.operations_migrations',
+    ),
+)
+class ContentTypeManagerOperationsTests(TransactionTestCase):
+    databases = {'default', 'other'}
+    available_apps = [
+        'contenttypes_tests',
+        'django.contrib.contenttypes',
+    ]
+
+    def setUp(self) -> None:
+        call_command('migrate', 'contenttypes_tests', 'zero', database='default', interactive=False, verbosity=0)
+
+    def test_get_for_models_creation_on_migration(self):
+        ContentType.objects.clear_cache()
+        call_command('migrate', 'contenttypes_tests', '0003', database='default', interactive=False, verbosity=0)
+
+    def test_get_for_models_load_on_migration(self):
+        ContentType.objects.create(app_label='contenttypes_tests', model='foo')
+        ContentType.objects.create(app_label='contenttypes_tests', model='baz')
+        ContentType.objects.clear_cache()
+        call_command('migrate', 'contenttypes_tests', '0003', database='default', interactive=False, verbosity=0)
```

Comment:
Can we check that `get_for_models()` works properly in migrations, not only confirm that it doesn't crash?